Localize/lang/strings.json

```
@@ -4,12 +4,9 @@
   "+20fdv": "Invalid properties '{0}' for authentication type '{1}'.",
   "+3efoo": "No schema is added",
   "+DxBpE": "Failed loading the schema. Please try again.",
-  "+JCU+Y": "Select this if you are configuring an on-prem connection",
-  "+W2vTp": "Please enter an e-mail address",
   "+W9fkq": "(UTC+08:00) Kuala Lumpur, Singapore",
+  "+ZSBrq": "Context menu for {title} card",
   "+dlOgj": "(UTC-05:00) Haiti",
-  "+g+mch": "Returns the body for a part in a multipart output of an action",
-  "+iMTt7": "Check to select this connection",
   "+jnjtp": "Select a Manual Workflow resource",
```

The original text is confusing. Do customers know what is a "manual workflow resource"? Does this really mean a workflow with a Request trigger, which is named 'manual' in code view?

```suggestion
  "+jnjtp": "Select workflow with 'manual' trigger",
```

---

@ccastrotrejo: Any way to get clarification about the meaning? Thanks!